de.metas.business.rest-api/src/main/java/de/metas/rest_api/ordercandidates/request/JsonOLCandCreateRequest.java

```diff
@@ -200,14 +202,29 @@
 	@JsonInclude(Include.NON_NULL)
 	private LocalDate presetDateShipped;
 
+	@ApiModelProperty(value = "A C_Payment entry will be created based on the info here")
+	private JsonPaymentInfo paymentInfo;
+
+	@ApiModelProperty(value = "Specifies if the created order will be a normal Sales Order or a Prepaid Sales Order")
```

Comment:
document which defaul value metasfrsh will pick if ommitted (should sales order)

Reply:
Done